{
	"schema_version": "1.6.0",
	"id": "RHSA-2019:3744",
	"modified": "2024-09-13T19:54:41Z",
	"published": "2024-09-13T19:54:41Z",
	"related": [
		"CVE-2019-10156",
		"CVE-2019-10206",
		"CVE-2019-3828"
	],
	"summary": "Red Hat Security Advisory: ansible security update",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.0/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:N"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:openstack:14::el7",
				"name": "ansible",
				"purl": "pkg:rpm/redhat/ansible"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:2.6.19-1.el7ae"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2019:3744.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2019:3744"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1676689"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1717311"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1732623"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2019/rhsa-2019_3744.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2019-3828"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2019-3828"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2019-3828"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/ansible/ansible/pull/52133"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2019-10156"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2019-10156"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2019-10156"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2019-10206"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2019-10206"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2019-10206"
		},
		{
			"type": "ARTICLE",
			"url": "https://github.com/ansible/ansible/pull/59246"
		}
	]
}
